src/ansi-c/c_qualifiers.cpp

```diff
@@ -9,6 +9,27 @@ Author: Daniel Kroening, kroening@kroening.com
 #include "c_qualifiers.h"
 
 #include <ostream>
+#include <util/make_unique.h>
+
+c_qualifierst &c_qualifierst::operator=(const c_qualifierst &other)
+{
+  is_constant = other.is_constant;
+  is_volatile = other.is_volatile;
+  is_restricted = other.is_restricted;
+  is_atomic = other.is_atomic;
+  is_noreturn = other.is_noreturn;
+  is_ptr32 = other.is_ptr32;
+  is_ptr64 = other.is_ptr64;
+  is_transparent_union = other.is_transparent_union;
+  return *this;
+}
+
+std::unique_ptr<c_qualifierst> c_qualifierst::clone() const
```

Comment:
What is this good for?

Reply:
Answered on your comment at the call site.